components/Markdown.php

```diff
@@ -1101,7 +1101,7 @@ public function _doHeaders_callback_setext( $matches ) {
 				return $matches[0];
 			}
 
-			$level = $matches[2]{0} == '=' ? 1 : 2;
+			$level = $matches[2][0] == '=' ? 1 : 2;
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 8e2ea1b807e3305396c2969c1f43418f</sub>